jupyterlab_bigquery/src/components/query_editor/query_text_editor/query_text_editor.tsx

```diff
@@ -595,7 +596,7 @@ class QueryTextEditor extends React.Component<
     const { iniQuery } = this.props;
 
     // eslint-disable-next-line no-extra-boolean-cast
-    const queryValue = !!iniQuery ? iniQuery : 'SELECT * FROM *';
+    const queryValue = !!iniQuery ? iniQuery : '';
```

Comment:
I think an empty editor works nicer than a `SELECT * FROM *` as Polong noted in a feedback comment:

> After installing the extension and clicking on the extension for the first time to see the console in a new tab, the default syntax is:
SELECT * FROM *
>
>It shows an error, which makes me feel like I've done something wrong. It should instead be blank or use a legitimate query

Reply:
An empty query sounds good to me